Bootstrapper1/Bootstrapper.wixproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="WixToolset.Sdk/4.0.0-rc.2">
```

Comment:
Please change the name of the folder from `Bootstrapper1` to `Bootstrapper` as well

Reply:
Or you could rename it to `MetaMorpheusBootstrapper` which would group it with the other wix project.

Reply:
I'll try grouping both, appreciate the feedback. :)